core/domain/suggestion_services.py

```diff
@@ -165,6 +166,18 @@ def _update_suggestion(suggestion):
     suggestion_model.put()
 
 
+def edit_suggestion(suggestion, change_object):
+    """Edits the submitted suggestion.
+
+    Args:
+        suggestion: obj. The suggestion to be changed.
```

Comment:
obj is not a data type. Specify the class the object is an instance of (look at other docstrings to get an idea).

Reply:
done